modules/tensor_mechanics/test/tests/rom_stress_update/gold/3d_out.csv

```diff
@@ -0,0 +1,7 @@
+time,effective_strain_avg,immobile_disloactions,mobile_dislocations,temperature
+0,0,0,0,0
+1,1.99257492218e-11,440000008780.87,5999998298848,900
+2,3.9851463024745e-11,440000017561.74,5999996597699,900
+3,5.9777141408846e-11,440000026342.62,5999994896553,900
+4,7.9702784373972e-11,440000035123.5,5999993195410.1,900
+5,9.9628391920457e-11,440000043904.37,5999991494270.2,900
```

Comment:
have these values been checked against the standalone rom too, to make sure that the values align? My original implementation had some typos in both the coefficients and in the unconvert methods--you probably caught them, so I'm just double checking

Reply:
No, 3d.i and 2drz.i are just dummy implementations. The standalone rom is tested in verification.i